packages/plugins/rubygem-smart_proxy_dynflow_core/rubygem-smart_proxy_dynflow_core.spec

```
@@ -33,6 +33,7 @@ Requires: %{?scl_prefix}rubygem(sinatra)
 Requires: %{?scl_prefix}rubygem(rack)
 Requires: %{?scl_prefix}rubygem(sqlite3)
 Requires: %{?scl_prefix}rubygem(logging)
+Requires: %{?scl_prefix}rubygem(logging-journald)
```

```suggestion
Requires: %{?scl_prefix}rubygem(logging-journald) >= 2.0
Requires: %{?scl_prefix}rubygem(logging-journald) < 3.0
```

See https://github.com/theforeman/smart_proxy_dynflow/blob/60eadab42a18a2ec5387c1bb85e1a70e270e077c/Gemfile#L18